docker-compose/docker-compose.yml

```diff
@@ -34,29 +36,37 @@ services:
             - 8082:8080 #Browser port
 
     alfresco-pdf-renderer:
-        image: alfresco/alfresco-pdf-renderer:1.1
+        image: alfresco/alfresco-pdf-renderer:1.2
```

Comment:
Why did the other 3 receive a version bump as part of this PR / set of commits ?

I guess I should review the PRs/commits in main code base first (and then packaging / deployment) ;-)

Reply:
The PO for the repo team wanted all the transformers lock stepped to together to have the same version, so when one is released they all are. Originally they all had separate release cycles. This may not be the case in the future.